tests/test_timereport.py

```diff
@@ -30,3 +30,15 @@ def test_factory():
     assert isinstance(test_data.get('event_date'), datetime.datetime)
     for item in ('user_id', 'user_name', 'reason', 'hours'):
         assert isinstance(item, str)
+
+
+def test_slack_token():
+    assert verify_token('faulty fake token') is not True
+    fake_test_token = 'my_fake_token'
+    os.environ['SLACK_TOKEN'] = fake_test_token
+    assert verify_token(fake_test_token) is True
+
+
+def test_verify_reason():
+    assert verify_reasons(['not real reasons'], 'fake') is not True
+    assert verify_reasons(['my fake reason'], 'my fake reason') is True
```

Comment:
should we use the actual values here for clarity? 

for example

```
verify_reasons(['vab'], 'vab') is True 
?
```

Reply:
I don't think so. The function is only there to validate any reason. Writing an opinionated reason in a test doesn't really make sense to me. I think it's more clear to have obvious test data to make the test even more verbose on what it's actually testing.

Just my opinion though. If you really want us to test a "real reason" then sure. But you will have to give me a good reason for it 😏